### ISSUE TITLE
nginx_webroot_location hardcoded to /var/www/html , fails with default_vhost=0

### ISSUE THREAD

**Issue by sshilko** [puphpet/puphpet#2093]
Ubuntu 14.04

when 
```
nginx:
    install: '1'
    settings:
        default_vhost: 0
```
setup fails because of depencency

```
Error: Could not find dependency File[/var/www/html] for Exec[Set /var/www/html/index.html contents] at /tmp/vagrant-puppet/manifests-75f35e3bc7e32744860c4bb229c88812/Nginx.pp:337
```

the 337 line:
```
   exec { "Set ${default_vhost_index_file} contents":
     command => "cat ${default_vhost_source_file} > ${default_vhost_index_file} && \
                 chmod 644 ${default_vhost_index_file} && \
                 chown root ${default_vhost_index_file} && \
                 chgrp ${webroot_group} ${default_vhost_index_file} && \
                 touch /.puphpet-stuff/default_vhost_index_file_set",
     returns => [0, 1],
     creates => '/.puphpet-stuff/default_vhost_index_file_set',
     require => File[$puphpet::params::nginx_webroot_location],
   }
```

because
puppet/modules/puphpet/manifests/params.pp
```
  $nginx_webroot_location = $::osfamily ? {
    'Debian' => '/var/www/html',
    'Redhat' => '/var/www/html'
  }
```

**Comment by head82** [comment]
+1
same problem

**Comment by sshilko** [comment]
i just commented out the whole exec {...} block and it works for me, as a workaround

**Comment by jtreminio** [comment]
fixed by #2082
